modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -1472,7 +1473,7 @@ Int2ObjectMap<RowBatch> toRowBatchByPartitionId(Collection<BinaryRowEx> rows) {
     }
 
     /** {@inheritDoc} */
-    // TODO: https://issues.apache.org/jira/browse/IGNITE-19619 The method should be removed, SQL engine should use placementDriver directly
+    // TODO: https://issues.apache.org/jira/browse/IGNITE-20701 The method should be removed
```

Comment:
```suggestion
    // TODO: https://issues.apache.org/jira/browse/IGNITE-20933 The method should be removed
```

I can't do it in #2825 - there are some other usages. Let's remove separately.

Reply:
@ptupitsyn **assignments** is used from 1 test in your #2825 PR, i fix this usage here, probably you can take this test fixture into your branch ? Or i can see - 2 approvals on your branch, if you merge it - i can easily fix this place.